recipe/meta.yaml

```diff
@@ -13,22 +13,25 @@ source:  # [win]
     fn: mingwpy-0.1.0b3-cp27-none-win_amd64.whl                                                       # [py27 and win64]
 
 build:
-    skip: True # [not win]
     number: 0
```

Comment:
You should probably bump the build number for a change of this size. Particularly as the runtime dependencies have changes (this is a functional change).